src/desktops/dskGameInterface.cpp

```diff
@@ -689,6 +689,9 @@ bool dskGameInterface::Msg_KeyDown(const KeyEvent& ke)
                 {
                     isCheatModeOn = !isCheatModeOn;
                     curCheatTxt.clear();
+                    char text[256];
+                    snprintf(text, sizeof(text), _("Cheat mode is %s"), isCheatModeOn ? _("on") : _("off"));
```

Comment:
i would here also prefer the `%`-operator instead the snprintf

Reply:
Don't use snprintf at all. Use Boost format which is safer. BTW: No need for this message. There already is an icon showing whether cheat mode is on